sys/net/gnrc/routing/ipv6_auto_subnets/gnrc_ipv6_auto_subnets.c

```diff
@@ -622,6 +634,10 @@ static void *_eventloop(void *arg)
     uint8_t idx_start = 0;
     uint8_t subnets = local_subnets;
     uint8_t tx_period = CONFIG_GNRC_IPV6_AUTO_SUBNETS_TX_PER_PERIOD;
+#if CONFIG_GNRC_IPV6_AUTO_SUBNETS_STATIC
+    uint8_t _idx_old = 0;
+    uint8_t _subnets_old = 0;
```

Comment:
The `_` prefix usually is used for static globals (copying somewhat the python paradigm to mark private members as such). So why is it used here?